docs/code.rst

```diff
@@ -536,6 +537,7 @@ Community contributed hooks
 .. autoclass:: airflow.contrib.hooks.vertica_hook.VerticaHook
 .. autoclass:: airflow.contrib.hooks.wasb_hook.WasbHook
 .. autoclass:: airflow.contrib.hooks.winrm_hook.WinRMHook
+.. autoclass:: airflow.contrib.hooks.telegram_hook.TelegramHook
```

Comment:
Please add the class in alphabetical order. First, according to class names, and then according to the module path.